samples/TestApp.PCL/TestApp.PCL.csproj

```diff
@@ -1,6 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <TargetFramework>netstandard1.1</TargetFramework>
+	<DebugType>Full</DebugType>
```

Comment:
Support for PPDB won't ship with RTW, so this is the only way for the time being, I'm afraid.

We may have an alpha soon-ish, but without this, you just can't build.